db/seeds.rb

```diff
@@ -68,9 +68,10 @@
     user = User.create!(
       name: name,
       profile_image: File.open(Rails.root.join("app/assets/images/#{rand(1..40)}.png")),
-      twitter_username: Faker::Internet.username(specifier: username),
+      # Twitter username should be always ASCII
+      twitter_username: Faker::Internet.username(specifier: username.transliterate),
       # Emails limited to 50 characters
-      email: Faker::Internet.email(name: username, separators: "+", domain: Faker::Internet.domain_word.first(20)),
+      email: Faker::Internet.email(name: username.transliterate, separators: "+", domain: Faker::Internet.domain_word.first(20)),
```

Comment:
**Addition**: made it compatible with #16115, to keep the username relevant to the personal name while ensuring ASCII. It uses a function from `sterile`, which I don't generally prefer for transliteration, but okay since it is not seriously meant to make sense here.

Reply:
Can you please address the Rubocop error?